### ISSUE TITLE
[Bug]: v4.8.69 update to 3.x canvas - Cannot assign to read only property 

### ISSUE THREAD

**Issue by prescottprue** [mozilla/pdf.js#19046]
### Attach (recommended) or Link to PDF file

N/A

### Web browser and its version

JSDom 25.0.1, Chrome 131.0.6778.70

### Operating system and its version

Ubuntu 22

### PDF.js version

v4.8.69

### Is the bug present in the latest PDF.js version?

Yes

### Is a browser extension

No

### Steps to reproduce the problem

1. Install v4.8.69 of pdfjs-dist
2. Attempt to use canvas API such as `canvas.getContext` ([as described in Mozilla MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API#javascript)):
    ```js
    const canvasCtx = document.createElement('canvas').getContext('2d')
     ```
3. See `Cannot assign to read only property` error such as `Cannot assign to read only property 'createPattern'  of object '#<CanvasRenderingContext2D'`

### What is the expected behavior?

Its expected that using the canvas API continues to work as [as described in Mozilla MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API#javascript) as it did in v4.7.76

### What went wrong?

v4.8.69 of pdfjs-dist updates to canvas v3.0.0 RC which has [a known issue in canvas with read only properties](https://github.com/Automattic/node-canvas/issues/2446) ([a PR has been merged to fix this](https://github.com/Automattic/node-canvas/pull/2409) but has not been released yet)

**Comment by Snuffleupagus** [comment]
Fixed by PR #19015.

**Comment by jacob-israel-turner** [comment]
Thanks for fixing this @Snuffleupagus! It'll be nice to not have to wrestle with canvas going forward.

Any idea when the next release will be cut? I'm still encountering this bug on v4.8.69, and I haven't found a workaround.
